x/webauthnx/js/webauthn.js

```diff
@@ -272,7 +272,12 @@
       })
       .catch((err) => {
         // Calling this again will enable the autocomplete once again.
-        console.error(err)
+        if (err instanceof DOMException && err.name === "SecurityError") {
+          console.error(`A security exception occurred while loading Passkeys / WebAuthn. To troubleshoot, please head over to https://www.ory.sh/docs/troubleshooting/passkeys-webauthn-security-error. The original error message is: ${err.message}`)
+        } else {
+          console.error(err)
```

Comment:
Maybe while we're here, add some explanation what happened here and why we got here with Ory context?

Reply:
What you mean?

Reply:
```suggestion
          console.error("[Ory/Passkey] An unknown error occurred while getting passkey credentials", err)
```